packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -114,75 +102,38 @@ const useBarStyles = makeStyles({
   },
 });
 
-const useTrackStyles = makeStyles({
-  base: {
-    gridRowStart: '2',
-    gridColumnStart: '1',
-    backgroundColor: tokens.colorNeutralBackground6,
-
-    '@media screen and (forced-colors: active)': {
-      ...shorthands.borderBottom('1px', 'solid', 'CanvasText'),
-    },
-  },
-  medium: {
-    height: barThicknessValues.medium,
-  },
-  large: {
-    height: barThicknessValues.large,
-  },
-});
-
 /**
  * Apply styling to the Progress slots based on the state
  */
 export const useProgressStyles_unstable = (state: ProgressState): ProgressState => {
-  const { indeterminate, thickness, percentComplete } = state;
+  const { max = 1.0, thickness, value } = state;
```

Comment:
I don't think you need to specify a default value here since you already did it in `useProgress`.

```suggestion
  const { max, thickness, value } = state;
```